nav2_dynamic_motion_model/package.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>nav2_dynamic_motion_model</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
```

Comment:
Consider what a package should be named if we want it to include the motion models and the util that uses them with track messages for predictions in one package.